applications/CoSimulationApplication/tests/fsi_wall/fsi_wall_cfd_results_disp_ref.dat

```diff
@@ -1,4 +1,4 @@
-# Results for "node" with Id # 577 at for search config "initial" position: x0: 9.5; y0: 6.03; z0: 0
+# Results for "node" with Id # 577 for search config "initial" at position: x0: 9.5; y0: 6.03; z0: 0
```

Comment:
@mpentek this took me quite some time to find
I would have expected everything but this :sweat_smile:

Reply:
Nice